src/container_runner.py

```diff
@@ -18,6 +18,32 @@
 DOCKER_DAEMON_CONFIG_PATH = Path("/etc/docker/daemon.json")
 
 
+def _try_set_proxy_settings():
+    """If Juju proxy environment variables are present, set procy environment variables and write Docker proxy settings to /etc/docker/daemon.json."""
```

Comment:
typo "procy"

Reply:
Ah, thanks! I'll fix that before I merge :slightly_smiling_face: